AuthenticatorShared/Core/Platform/Services/Stores/AppSettingsStore.swift

```diff
@@ -319,6 +341,12 @@ extension DefaultAppSettingsStore: AppSettingsStore {
         return .never
     }
 
+    func hasSyncedAccount(name: String) -> Bool {
+        let hashedData = SHA256.hash(data: Data(name.utf8))
+        let hashedName = hashedData.map { String(format: "%02hhx", $0) }.joined()
```

Comment:
♻️  Could this be an extension of `String` or some helper function to create the hashed string? I see this being the same in `setHasSyncedAccount`. No rush but wanted to point it out.

Reply:
Good call. I can throw that into one of my upcoming PRs. 👍